core/templates/pages/library-page/library-page.component.ts

```diff
@@ -476,6 +495,59 @@ export class LibraryPageComponent {
       });
   }
 
+  moveClassroomCarouselToPreviousSlide(): void {
+    this.classroomCarouselIndex -= 1;
+    this.classroomCarousel.prev();
+  }
+
+  moveClassroomCarouselToNextSlide(): void {
+    this.classroomCarouselIndex += 1;
+    this.classroomCarousel.next();
+  }
+
+  onClassroomNavigationIndicatorClicked(slideEvent: NgbSlideEvent): void {
+    // Extract numeric index from slide id (format: 'ngb-slide-{index}')
+    this.classroomCarouselIndex = parseInt(slideEvent.current.split('-')[2]);
+  }
+
+  getClassroomChunkIndices(length: number): number[] {
+    const chunkSize = 3;
+    const numChunks = Math.ceil(length / chunkSize);
+    return Array(numChunks)
+      .fill(0)
+      .map((_, index) => index);
+  }
+
+  getClassroomsForChunk(
+    classroomSummaries: ClassroomSummaryDict[],
+    chunkIndex: number
+  ): ClassroomSummaryDict[] {
+    const chunkSize = 3;
+    const start = chunkIndex * chunkSize;
+    const end = start + chunkSize;
+    return classroomSummaries.slice(start, end);
+  }
+
+  showNextClassroomChunkButton(): boolean {
```

Comment:
the verb in methods should describe what the method does. In this case, it's not show, it's more of a shouldShow

Reply:
My bad, Done.